src/main.rs

```diff
@@ -296,6 +340,28 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
             profitable_item.count, item_id
         );
 
+        let unknown_recipes: Vec<u32> = profitable_item.used_recipes.iter().filter_map(|(id, known)| {
+            if !*known {
+                Some(*id)
+            } else {
+                None
+            }
+        }).collect();
+        if unknown_recipes.len() > 0 {
+            println!(
+                "You {} craft this yet. Missing recipe id{}: {}",
```

Comment:
This might make more sense when an api key isn't being used:
```suggestion
                "You {} craft this yet. Requires recipe id{}: {}",
```